src/styles/utils/mixins.scss

```diff
@@ -0,0 +1,7 @@
+@mixin placement-center {
+  position: absolute;
+  left: 50%;
+  top: 50%;
+
+  transform: translate(-50%, -50%);
+}
```

Comment:
It's cool that you divided your scss into files and followed the previous advice from the mentors. 
I'm thoroughly delighted😀

little fix: also leave 1 line at the end of mixins.scss and variables.scss files, index.html